runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -464,6 +467,20 @@ public void onAnimationChanged(AnimationChanged event)
 	{
 		Actor actor = event.getActor();
 
+		if (config.showAbyssalSireStun()
```

Comment:
You should move this block after the actor != localPlayer

Reply:
If it was moved there, it wouldn't ever work.

Reply:
oh, right true, oops